### PR TITLE
updates-mentioned-in-notes

### PR DESCRIPTION
**Pull request by AnilMandavilli594**
Added FZ webpage
added the workflow diagram

### CODE REVIEW

File: hugo.toml
Comment (robertu94) [inline hugo.toml:1]: changing the baseURL like this will break the production site.

File: content/en/.DS_Store
Comment (robertu94) [inline content/en/.DS_Store:1]: Please don't commit .DS_Store entries, and add them to the gitignore.

**Review by robertu94 — CHANGES_REQUESTED** [review]
See comments

File: hugo.toml
Comment (AnilMandavilli594) [reply hugo.toml:1]: if i doesnt change the baseurl to "/", the page is not working in my local computer

Comment (AnilMandavilli594) [reply hugo.toml:1]: Do you have any suggestion about it.

Comment (dingwentao) [reply hugo.toml:1]: Can we ensure this PR will be merged before next Monday so that we can view the new updates in our weekly meeting?

**Pull request merged**